docs/index.md

```diff
@@ -3,14 +3,106 @@ title: "edi3 Inter Customs Ledger 1.0 Specification"
 specID: "icl/1"
 status: "![raw](http://rfc.unprotocols.org/spec:2/COSS/raw.svg)"
 editors: "[Chris Gough](https://github.com/monkeypants)"
-contributors: 
+contributors: "[Richard Spellman](https://github.com/arpentnoir)"
 ---
 
 ## Introduction
+International trade clearance procedures require a number of documents to be presented to satisfy the requirements of a variety of agreements. For example, in order for an importer to gain preferential tariff treatment under a Free Trade Agreement, the importer must present a valid Certificate of Origin (issued to the exporter) which states that the goods being imported qualify as originating in the country which is party to the agreement. These processes are managed largely by the transfer of paper documents which are subject to loss, alteration and forgery. 
+> FIXME
```

Comment:
Let's make these tickets rather than inline FIXME's or TODO's - maybe have TBD/TODO under blank headings but not for more detailed improvements

Reply:
done in my PR